build.sbt

```diff
@@ -2,8 +2,26 @@ lazy val root = (project in file("."))
   .enablePlugins(SbtPlugin)
   .settings(
     name := "sbt-test-report",
-    version := "0.26.0",
+    version := "0.27.0",
```

Comment:
For another day - would auto versioning via `sbt-auto-build` not work rather than manual bumps?

Reply:
Yeah, will create a tech debt ticket for that 😅